rmgpy/tools/canteramodel.py

```diff
@@ -240,19 +268,21 @@ def generate_conditions(self, reactor_type_list, reaction_time_list, mol_frac_li
         `reaction_time_list`      A tuple object giving the ([list of reaction times], units)
         `mol_frac_list`           A list of molfrac dictionaries with species object keys
                                and mole fraction values
+        `surface_mol_frac_list`   A list of molfrac dictionaries with surface species object keys and mole fraction values
         To specify the system for an ideal gas, you must define 2 of the following 3 parameters:
         `T0List`                A tuple giving the ([list of initial temperatures], units)
         'P0List'                A tuple giving the ([list of initial pressures], units)
         'V0List'                A tuple giving the ([list of initial specific volumes], units)
         """
 
-        self.conditions = generate_cantera_conditions(reactor_type_list, reaction_time_list, mol_frac_list, Tlist, Plist, Vlist)
+        self.conditions = generate_cantera_conditions(reactor_type_list, reaction_time_list, mol_frac_list, surface_mol_frac_list, Tlist, Plist, Vlist)
 
     def load_model(self):
         """
         Load a cantera Solution model from the job's own species_list and reaction_list attributes
         """
-
+        # TODO add surface version - this is limited by the rmgpy.kinetics.arrhenius.Arrhenius.to_cantera_kinetics()
+        # function not being able to handle surface reactions yet
```

Comment:
Do we need a new issue to track this?

Reply:
Created this one https://github.com/ReactionMechanismGenerator/RMG-Py/issues/2573
Will delete the lines in the code.